### PR TITLE
Features/images categories

### PR DESCRIPTION
**Pull request by danyball**

### CODE REVIEW

**Comment by raphaellueckl** [comment]
All items don't have a width and height and the fill is set to `none`. Was that on purpose? Because we usually use `fill="currentColor"`.

File: src/components/00-materials/scripts/export-svg.js [hunk 1]
```diff
@@ -14,80 +14,84 @@ const camelCase = _camelCase =>
...
+        const contents = fs
+          .readFileSync(iconPath, 'utf8')
+          .replace(/(\r\n|\n|\r)/gm, '');
+
+        let fileName = path.basename(icon);
+        fileName = cleanFileName(fileName);
+        const className = toClassName(fileName);
+
+        // Generate a js file forEach svg file found
+        fs.writeFileSync(
```
Comment (raphaellueckl) [inline src/components/00-materials/scripts/export-svg.js]: It doesn't make sesne to use synced methods here. But I leave it up to you to correct that, sicne I'm not sure how often we use that.

**Comment by raphaellueckl** [comment]
Why is the script not recursive?

**Comment by danyball** [comment]
> Why is the script not recursive?

Its not that easy with `fs` and we do not need it for now.

File: src/components/00-materials/scripts/export-svg.js (hunk 1 above)
Comment (danyball) [reply src/components/00-materials/scripts/export-svg.js]: :D

**Comment by danyball** [comment]
> All items don't have a width and height and the fill is set to `none`. Was that on purpose? Because we usually use `fill="currentColor"`.

For images these attributes are not relevant for now.

**Comment by MKaHead** [comment]
- recursive script nice to have, but we should enable the images. We can open a new feature issue for the script.
- the images don't need current color because they have fixed color

**Pull request merged**